src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -414,6 +414,8 @@ void uct_rc_mlx5_iface_common_tag_cleanup(uct_rc_mlx5_iface_common_t *iface)
 {
     if (UCT_RC_MLX5_TM_ENABLED(iface)) {
         if (UCT_RC_MLX5_MP_ENABLED(iface)) {
+            kh_destroy_inplace(uct_rc_mlx5_mp_hash_lid, &iface->tm.mp.hash_lid);
```

Comment:
need to check if hashes are empty, if not print some warning or at least release remaining descriptors

Reply:
what do you mean by release descriptors? The memory will be released by `kh_destroy_inplace`
I'd print another warning to avoid flooding like about requests which are not returned to mpool

Reply:
if the hash is non-empty, isn't this a potential bug? need to print warning in such case

Reply:
it means some message has not arrived completely. Seems like can happen during force close